src/ui/panels/clan.js

```diff
@@ -99,20 +103,28 @@ export class Clan {
     }
 
     // Add clan member to list
-    addClanMember(member) {
+    addClanMember(memberName) {
         // Add to saved data
-        characterData.addClanMember(member.name);
+        characterData.addClanMember(memberName);
 
         // Add visuals if the dashboard is up
         if (this.dashboard.scene.isActive()) {
             const startX = 560,
                 startY = 280,
                 yDiff = 18;
 
-            // Add text to the list
-            let index = this.clanMembers.length;
-            member.createText(false, startX, startY + index * yDiff);
-            this.clanMembers.push(member);
+            // Add new clan entry if new member else increment existing members
+            const member = this.clanMembers.find((member) => member.name === memberName);
+            if (!member) {
+                member = getAutoclickerClass(memberName, this.scrollWindow);
```

Comment:
Declared as const but then it's overwritten, see my previous comment. This either needs to be changed to be
```js
const existingMember = this.clanMembers.find((member) => member.name === memberName);
if (!existingMember ) {
  const newMember = ...
  ...
  newMember.numberOwned = 1;
}
else {
  existingMember .numberOwned++;
}
```

or

```js
let member = this.clanMembers.find((member) => member.name === memberName);
if (!member ) {
  member = ...
  ...
}
member .numberOwned++;
```